### PR TITLE
Release-3.0-Dev-Engine

### PR DESCRIPTION
**Pull request by resmaa**
**Release 3.0.0**

Palette Dev Engine

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/916#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04.5-devx.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx.md:62]: Since they are by default enabled, may be it can reside into some other page with heading `Disable Social Sign In`

Comment (resmaa) [reply content/docs/04.5-devx.md:62]: sure @tiwarisumit3

File: content/docs/04.5-devx/01-dev-land-explore.md
Comment (tiwarisumit3) [inline content/docs/04.5-devx/01-dev-land-explore.md:25]: no need to mention for app modeling again. Also our quota we can speak up here or link to the quota what we are providing

File: content/docs/04.5-devx.md
Comment (truegilb) [inline content/docs/04.5-devx.md:29]: Using the App Profile to deploy your application

File: content/docs/04.5-devx.md
Comment (truegilb) [inline content/docs/04.5-devx.md:17]: Palette now makes it easy for app developers to model and deploy cloud-native applications at scale. <-- this is simpler. PDE is the marketing term and not a persona name.

What are the two wide perspectives?

**Review by truegilb — CHANGES_REQUESTED** [review]

File: content/docs/04.5-devx.md
Comment (truegilb) [inline content/docs/04.5-devx.md:42]: Palette provides two options for developers to log in...

File: content/docs/04.5-devx.md
Comment (truegilb) [inline content/docs/04.5-devx.md:67]: Because this page is focusing on "Quick start with Palette Dev Engine" - meaning helping users realizing the benefits such as free compute and fast app deploy, let's put the "disable social" to the administrative section.

I also think we should mention the free compute in the Quick Start page.

File: content/docs/04.5-devx/01-dev-land-explore.md
Comment (truegilb) [reply content/docs/04.5-devx/01-dev-land-explore.md:25]: Let's make it clear what free is about : “We are offering 12 vCPU, 16 GiB of memory, and 20 GiB of storage for free”

File: content/docs/04.5-devx/01-dev-land-explore.md
Comment (truegilb) [inline content/docs/04.5-devx/01-dev-land-explore.md:43]: Not "Create Custom Registries" - We can "Connect Custom Registries"

**Review by truegilb — CHANGES_REQUESTED** [review]
Some more comments.

File: content/docs/04.5-devx.md
Comment (resmaa) [reply content/docs/04.5-devx.md:18]: added 
* App Mode: The developer deploys applications rapidly.

* Cluster Mode: To deploy and manage Palette clusters and explore the devops persona of Palette.

File: content/docs/04.5-devx.md
Comment (resmaa) [reply content/docs/04.5-devx.md:43]: done

**Review by truegilb — CHANGES_REQUESTED** [review]
Please rename MD file names to sandbox-clusters. We have published the URL in promotional materials: [https://docs.spectrocloud.com/devx/sandbox-clusters]

**Review by truegilb — APPROVED** [review]
Let's approve this at this state and invite wider audience for review in docs-latest @resmaa

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/916#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/916#pullrequestreview-1148420719" title="Approved">truegilb</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
